packages/suite-desktop/src/libs/user-data.ts

```diff
@@ -73,6 +73,28 @@ export const read = async (directory: string, name: string): Promise<InvokeResul
     }
 };
 
+export const readDir = async (directory: string): Promise<InvokeResult<string[]>> => {
+    const dir = path.join(app.getPath('userData'), directory);
+
+    try {
+        await fs.promises.access(dir, fs.constants.R_OK);
+    } catch {
+        await fs.promises.mkdir(dir);
+
+        return { success: true, payload: [] };
+    }
+
+    try {
+        const dirFiles = await fs.promises.readdir(dir);
+        const filteredDirFiles = dirFiles.filter(file => !/^\./.test(file));
```

Comment:
nit: I'd maybe go with `!startsWith('.')` to make it readable also for folks who are not into regexps. feel free to dismiss this nit